packages/react/CHANGELOG.md

```diff
@@ -3,51 +3,112 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
-## [2.147.4](https://github.com/carbon-design-system/carbon-addons-iot-react/compare/v2.147.3...v2.147.4) (2022-01-31)
```

Comment:
I think some of these merge conflicts we need to keep both versions and re-arrange them into the correct order. `master` CHANGELOG.md currently, correctly shows the 8.7 fixes that we merged directly into master. Your changes for the [hidden row action](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/master/CHANGELOG.md#21474-2022-01-31) for example that are being removed in the diff here (and following lines). However, I believe in this conflict both changes should have been kept, with the 2.147.4 changes being moved below all these other new additions of `2.148.0-next.X`, so that the changelog continues to correctly show the features/fixes we merged directly into `master`, but they'll be lower down the changelog file (as compared to currently at the top of the master branch's CHANGELOG), and the new `next` CHANGELOG entries (2.148.0-next.0 and newer) should now be at the top.

Reply:
Yes good catch thanks, I was a bit confused about this. I will fix it.